requirements-optional.txt

```diff
@@ -36,6 +36,8 @@ m2cgen>=0.9.0  # For model conversion
 evidently~=0.4.16
 
 # Parallel
+dask>=2024.4.1            # Python 3.11.9 breaks Dask (https://github.com/dask/dask/issues/11038)
+distributed>=2024.4.1     # Python 3.11.9 breaks Dask (https://github.com/dask/dask/issues/11038)
 fugue
 flask
 Werkzeug>=2.2,<3.0
```

Comment:
> Previous versions of Dask, which is pulled in transitively via Fugue through `pycaret[parallel]`, stopped working on Python 3.11.9, so that needs an update.

Of course, this should actually go into Fugue, but can also be fixed on both projects, depending on how fast this would converge into corresponding bugfix releases on the one project, or the other, and how convenient package maintainers would like to have the installation process for their users.

So, in the same spirit, that patch submits the same updates to the Fugue project.

- https://github.com/fugue-project/fugue/pull/537